src/Util/CompositionUtil.ts

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { Compositions } from '../Component/CompositionContext/CompositionContext';
+const _get = require('lodash/get');
+
+export interface CompositionUtilOptions {
+  composition: Compositions;
+  path: string;
+  propName: string;
+  propValue: any;
+  defaultElement: React.ReactElement;
+  onChange: Function;
+  onChangeName?: string;
+}
+
+interface CompositionUtilInjectOptions {
+  component: React.ReactElement;
+  onChange: Function;
+  propName: string;
+  propValue: any;
+  onChangeName?: string;
+}
+
+class CompositionUtil {
```

Comment:
Please add code docs.

Reply:
solved in https://github.com/geostyler/geostyler/pull/1186/commits/587b9b93e34174a44cce9a8464bcb50512687670